modules/actions/pool/v3/sync-hook-data.ts

```diff
@@ -38,7 +38,7 @@ export const syncHookData = async (
         // Get hooks data
         const data = await fetchHookData(viemClient, hookData.address, hookType, pool.address);
 
-        const name = `${hookType.charAt(0).toUpperCase()}${hookType.slice(1)}`;
+        const name = `${hookType.charAt(0).toUpperCase()}${hookType.slice(1, -4)}`;
```

Comment:
Concrete example:
- Hook name on API -> StableSurgeHook
- Hook name as balancer maths expect -> StableSurge

As mentioned on slack - I'd like feedback on this workaround.
Should we update hook names to remove `Hook` suffix on DB as well?